### DISCUSSION TITLE
Access to apk files and receive a direct link.

### DISCUSSION THREAD

**Discussion by Maximoff** [rumboalla/apkupdater#482] [Ideas]
Hello. The program really lacks direct access to downloaded apk files. When updating, the signature of the downloaded apk may often not match the signature of the already installed one (for FDroid this is often the case, for example). As a result, the application cannot be installed, and nothing can be done about it (I sometimes sign such applications with my signature in order to update from different sources). I suggest adding the ability to change the folder for downloaded files, with the ability to save them to internal memory. Or make it possible to send a direct link to download the file to a third-party downloader. Or better yet, do both. Thank you 😊

**Comment by accx12** [comment]
That would be a really helpful update, especially for users who often face signature issues while trying to install apk files. Having better control over file storage can make things smoother—just like when you [download bus simulator indonesia mod apk unlimited money](https://bussimulatorindo.com/), it's easier if files are saved where you want.
